README.md

```diff
@@ -745,6 +745,16 @@ class AddAvatarFingerprintColumnToUser < ActiveRecord::Migration
 end
 ```
 
+The algorithm can be specified using a configuration option; it defaults to MD5.
```

Comment:
I'd add here "for backwards compatibility", so people know we are working on providing better defaults already.